nginx/proxy.conf

```diff
@@ -1,3 +1,5 @@
+resolver 127.0.0.1;
```

Comment:
Yes it is. As the hostname used in `proxy_pass` is no more static but dynamic, nginx needs to know which DNS resolver to connect to to do the resolution.